models/ConfigureForm.php

```diff
@@ -34,6 +34,7 @@ public function attributeLabels()
         return [
             'enabled' => Yii::t('AuthOrcidModule.base', 'Enabled'),
             'clientId' => Yii::t('AuthOrcidModule.base', 'ORCID Client ID'),
+            'orcidAttribute' => Yii::t('AuthOrcidModule.base', 'Internal name of profile attribute to hold ORCIDs. If not entered, {orcidAttribute} will be used.', ['orcidAttribute' => Html::tag('code', 'orcid')]),

```

Comment:
I'm not sure that `attributeLabels()` would allow for this as it would render as plaintext, also you're applying a Html class without a use statement which would throw an error.

```txt
Internal name of profile attribute to hold ORCIDs. If not entered, <code>orcid</code> will be used.
```

Just use `['orcidAttribute' => 'orcid']` and it should be fine